rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -397,6 +397,18 @@ void main_window::InstallPackages(QStringList file_paths, bool show_confirm)
 		}
 	}
 
+	if (!file_paths.isEmpty())
+	{
+		// Handle the actual installations with a timeout. Otherwise the source explorer instance is not usable during the following file processing.
+		QTimer::singleShot(100, [this, file_paths]()
```

Comment:
Have you tried setting the delay to 0 ms? It should make it fire as soon as the current function returns. I don't see any reason to specifically wait 100 milliseconds.

Reply:
No. Die you?

Reply:
Haven't had time to test it, but I thought i'd give a suggestion on how to improve this. No need to get offended.

Reply:
what do you mean

Reply:
I just wanted to know if it works before I change it. But I tried later and it did.

Reply:
Oh sorry, thought you were being snarky because I left it as a comment without having tested it myself.